### PR TITLE
Update azure-ad-ipv6-support.md

### PR DESCRIPTION
**Pull request by Abizerh**
"How to find the IPv6 address used to sign-in to Azure AD" section added by Abizerh

### CODE REVIEW

File: support/azure/active-directory/azure-ad-ipv6-support.md [hunk 1]
```diff
@@ -65,6 +65,68 @@ For the IP ranges that are specified for Azure AD, make sure that you allow outb
...
 By default, both IPv6 and IPv4 traffic is supported on Windows and most other operating system (OS) platforms. Changes to the standard IPv6 configuration may result in unintended consequences. For more information, see [Guidance for configuring IPv6 in Windows for advanced users](/troubleshoot/windows-server/networking/configure-ipv6-in-windows).
 
+## How to find the IPv6 address used to sign-in to Azure AD
+To help admins find the IPv6 addresses used to sign-in to Azure AD, either during proactive testing 
+[Name Resolution Policy Table (NRPT)](https://learn.microsoft.com/en-us/troubleshoot/azure/active-directory/azure-ad-ipv6-support#test-azure-ad-authentication-over-ipv6) mentioned above OR post the IPv6 enablement in March 2023, use the methods below. Finding the top IPv6 addresses that are owned and managed by the organization will help you add the same under ‘Named location”.
+
+### Methods to get the list of IPv6 addresses from Sign-in logs:
+
+   #### Filtering Azure AD Sign-in logs
+   - In the Azure portal, search for and select Azure Active Directory.
```
Comment (MicrosoftGuyJFlo) [inline support/azure/active-directory/azure-ad-ipv6-support.md:75]: What role is needed?

File: support/azure/active-directory/azure-ad-ipv6-support.md [hunk 2]
```diff
@@ -65,6 +65,68 @@ For the IP ranges that are specified for Azure AD, make sure that you allow outb
...
+   -	Azure AD logs
+   -	Azure resources logs
+   -	Logs from independent software tools
+
+   Note
+   You need an Azure AD Premium license to export reporting data to Azure Monitor.
+
+   To query for legacy TLS entries using Azure Monitor:
+
+   •	In Integrate Azure AD logs with Azure Monitor logs, follow the instructions for how to access the Azure AD sign-in logs in Azure Monitor.
```
Comment (MicrosoftGuyJFlo) [inline support/azure/active-directory/azure-ad-ipv6-support.md:97]: Where is this?

File: support/azure/active-directory/azure-ad-ipv6-support.md [hunk 3]
```diff
@@ -65,6 +65,68 @@ For the IP ranges that are specified for Azure AD, make sure that you allow outb
...
+ 
+
+        union SigninLogs, AADNonInteractiveUserSignInLogs
+        | where IPAddress has ":"
+        | summarize RequestCount = count() by IPAddress, AppDisplayName, NetworkLocationDetails
+        | sort by RequestCount
+
+   •	Select Run to execute the query. The log entries that match the query appear in the Results tab below the query definition.
+
+   #### Exporting logs through PowerShell
```
Comment (MicrosoftGuyJFlo) [inline support/azure/active-directory/azure-ad-ipv6-support.md:109]: How do I run this? You mention some permissions partway through the script.

**Review by MicrosoftGuyJFlo — CHANGES_REQUESTED** [review]
Needs some more info before I can even edit. I can go run down all of this but I dont have time today so it would be late next week at the earliest without more details or links provided.

File: support/azure/active-directory/azure-ad-ipv6-support.md (hunk 1 above)
Comment (gumauler) [reply support/azure/active-directory/azure-ad-ipv6-support.md:75]: To access the sign-ins log for a tenant, you must have one of the following roles:

Global Administrator
Security Administrator
Security Reader
Global Reader
Reports Reader

File: support/azure/active-directory/azure-ad-ipv6-support.md (hunk 2 above)
Comment (gumauler) [reply support/azure/active-directory/azure-ad-ipv6-support.md:97]: it's missing this link Integrate Azure AD logs with Azure Monitor logs - https://learn.microsoft.com/en-us/azure/active-directory/reports-monitoring/howto-integrate-activity-logs-with-log-analytics

File: support/azure/active-directory/azure-ad-ipv6-support.md (hunk 3 above)
Comment (gumauler) [reply support/azure/active-directory/azure-ad-ipv6-support.md:109]: it needs to be similar to this https://learn.microsoft.com/en-us/troubleshoot/azure/active-directory/enable-support-tls-environment?tabs=powershell#overview-of-new-telemetry-in-the-sign-in-logs

Use PowerShell to filter and export sign-in log entries where legacy TLS is used.

Note
You need an Azure AD Premium license to call the [Microsoft Graph API](https://learn.microsoft.com/en-us/graph/use-the-api) through PowerShell.

To filter and export the sign-in log entries:

Using the Run as administrator option, open Windows PowerShell from the Start menu.

Run the following commands to install the [Microsoft Graph SDKs](https://learn.microsoft.com/en-us/graph/sdks/sdk-installation) and set the execution policy:

PowerShell

Copy
Install-Module Microsoft.Graph -Scope AllUsers
Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope CurrentUser
Save the following script to a PowerShell script (.ps1) file.

**Comment by AmandaAZ** [comment]
@MicrosoftGuyJFlo Let me know if you finish your review for this PR. Then, we will proceed to review and merge the changes.

**Pull request closed without merging**